### DISCUSSION TITLE
Where do I find the plugins mentioned in the guide?

### DISCUSSION THREAD

**Discussion by Oli72** [xournalpp/xournalpp#2580] [Q&A]
The [guide](https://xournalpp.github.io/guide/plugins/#notable-plugins) mentions several "notable plugins". Where do I find them for download?

**Comment by rolandlo** [comment]
Most of these plugins are only available in version 1.1.0+dev. They will not work on versions 1.0.x. So make sure you have version 1.1.0+dev installed. You can download that version from the [release page](https://github.com/xournalpp/xournalpp/releases/tag/nightly)

The plugins should be automatically installed in Xournal++'s [plugin folder](https://xournalpp.github.io/guide/plugins/#installation-folder). I think the Windows installer currently does not install the plugin folder. You can download them from the [repository plugin folder](https://github.com/xournalpp/xournalpp/tree/master/plugins)
